llvm/include/llvm/Support/DXILABI.h

```diff
@@ -99,6 +99,12 @@ enum class SamplerFeedbackType : uint32_t {
 const unsigned MinWaveSize = 4;
 const unsigned MaxWaveSize = 128;
 
+// D3D_ROOT_SIGNATURE_VERSION
+enum class RootSignatureVersion {
+  rootsig_1_0 = 0x1,
+  rootsig_1_1 = 0x2,
+};
```

Comment:
Should these be defined in BinaryFormat/DXContainer.h with the other d3d12 constants? They don't seem like they belong in DXILABI to me.

Also the naming here feels a bit awkward - we're kind of repeating ourselves with `RootSignatureVersion::rootsig_1_0` - Should it just be `RootSignatureVersion::V1_0` or something like that?